### ISSUE TITLE
Weapon Materials of 2.0 does not return weapon list

### ISSUE THREAD

**Issue by vj-liboon** [theBowja/genshin-db#35]
```
var genshinDb = require("genshin-db")

console.log(genshinDb.weapons("narukami", {matchCategories:true}))
```

Return: `undefined`

Also with _Mask_ and _Branch of Distant Sea_

**Comment by theBowja** [comment]
Good catch. Looks like I forgot to add the upgrade costs to the searchable categories. I will fix this today.

**Comment by theBowja** [comment]
fixed with commit bf901b08afdb16e027fb49ad9de4d1cf6c27c594
available in v3.9.3
example:
<details>
<summary>genshindb.characters('<b>arrowhead', { matchCategories: true }</b>)</summary>

```js
[ 'Amber', 'Diona', 'Fischl' ]
```

</details>
